pkg/lib/cockpit-po-plugin.js

```diff
@@ -91,6 +89,9 @@ module.exports = class {
                     if (!references.some(str => str.startsWith(`pkg/${this.subdir}`)))
                         continue;
 
+                    if (references.every(filter))
```

Comment:
I feel like this is the wrong way around. It's apparently consistent (the caller also specifies the "correct" filter), but it'd be more natural to understand if "filter" gets you the things you want, instead of the things you don't want.

I'm trying to validate this in my head what happens if a string has a reference in boh a .jsx and the .manifest file -- then it should appear in both, right? An `any(positive_filter)` would make this clearer.